build.cake

```diff
@@ -97,7 +109,17 @@ Task("Test")
         },
         ToolPath = "./tools/JetBrains.dotMemoryUnit.3.0.20171219.105559/lib/tools/dotMemoryUnit.exe"
     });
-});
+}
+void RunTestsWithoutProfiling() {
+    DotNetCoreTest("./source/Halibut.Tests/Halibut.Tests.csproj", new DotNetCoreTestSettings
+    {
+        ArgumentCustomization = args => {
+            args.Append("--configuration=" + configuration);

```

Comment:
I thought `configuration` is an argument of `DotNetCoreTestSettings` ?

Reply:
Ah good pickup thanks